app/models/appointment.rb

```diff
@@ -1,3 +1,6 @@
 class Appointment < ActiveRecord::Base
   belongs_to :user
+
+  validates_presence_of :summary
+  validates_presence_of :description
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.